pkg/apis/kubermatic/v1/etcd_restore.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	// EtcdRestorePhaseCompleted value indicating that the old Etcd statefulset has completed successfully.
 	EtcdRestorePhaseCompleted EtcdRestorePhase = "Completed"
+
+	// EtcdRestorePhaseEtcdLauncherNotEnabled value indicating that etcd-launcher is not enabled.
+	EtcdRestorePhaseEtcdLauncherNotEnabled EtcdRestorePhase = "EtcdRestorePhaseEtcdLauncherNotEnabled"
 )
 
 // +kubebuilder:validation:Enum=Started;StsRebuilding;Completed
```

Comment:
This needs to be updated to include the new phase. Will probably require updating the CRD codegen as well.